docs/_data/sidebar.yml

```diff
@@ -19,6 +19,40 @@ main:
       evaluate-conjur:
         title: Evaluate Conjur
         path: /get-started/evaluate-conjur.html
+  integrations:
```

Comment:
Can we put some hierarchy in this menu? Right now it's:

* Kubernetes (a demo)
* Ansible (a platform integration)
* Puppet (a platform integration)
* Jenkins (a demo)
* Chef (a demo)
* Summon (an ops tool)
* Go (an API client)
* Ruby (an API client)
* .NET (an API client)
* Java (an API client)

I propose we clarify by grouping them like such:

* Demos
  * Kubernetes
  * Jenkins
  * Chef
* Platforms
  * Ansible
  * Puppet
* Tools
  * Summon
* API Clients
  * Go
  * Ruby
  * .NET
  * Java

Reply:
Yup! This is on deck for the next iteration of the nav. We need to do some rework to how the nav is built to organize the nav like this, so in the spirit of small-batches, we just have them listed in priority/popularity for this.